packages/server/README.md

```diff
@@ -49,3 +49,57 @@ const { jwt, userId } = await crossmintAuth.getSession({
 });
 ```
 
+3. Store the authentication material in cookies
+
+If you are using a framework with access to the response object, you can store the authentication material in cookies by passing the response object to the `getSession` method:
+
+```ts
+const { jwt, userId } = await crossmintAuth.getSession(request, response);
+```
+
+## Set up custom refresh route
+
+To set up a custom refresh route, you can use the `handleCustomRefresh` method. This method will refresh the token and return the new authentication material. This way, the authentication material can be stored in cookies that are tied to the domain of the provided route.
+
+In environments that use the Fetch API for `Request` and `Response` objects, `handleCustomRefresh` will return the response object:
+
+```ts
+const response = await crossmintAuth.handleCustomRefresh(request);
+return response;
+```
+
+In environments that use Node.js API, you also need to provide the response object:
+
+```ts
+await crossmintAuth.handleCustomRefresh(req, res);
+res.end();
+```
+
+## Security
+
+The SDK allows you to set the `httpOnly`, `secure`, `domain` and `sameSite` options for the cookies. This way, you can control how the cookies are stored and transmitted. Putting this together with a custom refresh route, you can store the authentication material in HttpOnly cookies that are tied to the domain of the provided route.
+
+Configure CrossmintAuth to do so when creating the custom refresh route:
+
+```ts
+const crossmintAuth = CrossmintAuth.from(crossmint, {
+    cookieOptions: {
+        httpOnly: true,
+        sameSite: "Strict",
+        secure: true,
+        domain: ".example.com",
+    },
+});
+```
+
+## Advanced Usaged
```

Comment:
Why would this section be advanced?
I thought the entire point of this was to allow a custom refresh route that can store the cookie as httpOnly

Reply:
It's a more complicated setup, but I see what you mean, I'll change it to a more enticing title